joomla-projects/privacy-framework
# Improve login action log

Pull Request for Issue #136 .

### Summary of Changes
This PR makes some improvements to log action login feature:

- Use onUserAfterLogin instead of  onUserLogin event trigger to log success login action
- Implement onUserLoginFailure to log failed login action

### Testing Instructions
- Apply path
- Try to login using a valid username and password, check action log and make sure the login action is logged
- Try to login using a valid username and wrong password, check action log and make sure there is action is logged (Example User admin tried to login to site)
